service/src/main/java/org/ehrbase/tenant/TenantAspect.java

```diff
@@ -28,15 +29,14 @@
 import org.aspectj.lang.annotation.Pointcut;
 import org.aspectj.lang.reflect.MethodSignature;
 import org.ehrbase.api.annotations.TenantAware;
-import org.ehrbase.api.tenant.ExtractionStrategyAware;
 import org.ehrbase.api.tenant.TenantAuthentication;
 import org.ehrbase.api.tenant.TenantIdExtractionStrategy;
 import org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint;
 import org.springframework.security.core.context.SecurityContext;
 import org.springframework.security.core.context.SecurityContextHolder;
 
 @Aspect
-public class TenantAspect implements ExtractionStrategyAware {
+public class TenantAspect implements org.ehrbase.api.aspect.TenantAspect {
```

Comment:
Interface and implementation class should not have the same name. Since this is the default impl for the TenantAspect i suggest renaming this class to DefaultTenantAspect or something like that.

Reply:
Renamed it to Default......